edenai_apis/apis/corticalio/client.py

```diff
@@ -0,0 +1,49 @@
+import json
+from typing import Dict
+
+import requests as requests
+from requests import Response
+
+from edenai_apis.utils.exception import ProviderException
+
+
+class CorticalClient:
+    """
+    REST client for the Cortical.io NLP API
+    """
+
+    def __init__(self, api_settings: Dict) -> None:
+        self.api_settings = api_settings
+        self.auth_headers = {"Authorization": self.api_settings.get("api_key")}
+        self.base_url = self.api_settings.get("base_url").rstrip('/')
+
+    def extract_keywords(self, text: str, language: str):
```

Comment:
```suggestion
    def extract_keywords(self, text: str, language: Optional[str]):
```
Are you sure that it works like this if language is optional?

Reply:
we should default to english irrc?